### PR TITLE
logging resolved doc

### PR DESCRIPTION
**Pull request by cl0ete**
Add logging

### CODE REVIEW

File: aries_cloudagent/connections/base_manager.py [hunk 1]
```diff
@@ -254,6 +254,7 @@ async def resolve_invitation(
         try:
             doc_dict: dict = await resolver.resolve(self._profile, did, service_accept)
             doc: ResolvedDocument = pydid.deserialize_document(doc_dict, strict=True)
+            self._logger.warning("Resolved doc ===> ", doc)
```
Comment (dbluhm) [inline aries_cloudagent/connections/base_manager.py:257]: `warning` is a curios log level to choose for this; I think I'd recommend info or debug. Is there a reason you selected `warning` specifically?

**Review by dbluhm — COMMENTED** [review]
Looks like there's some DCO issues.

Hyperledger requires each commit be signed off. From the details on the failed check:

> DCO
> 
> There are 3 commits incorrectly signed off. This means that the author(s) of these commits failed to include a Signed-off-by line in their commit message.
> 
> To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).
> 
> Here is how to fix the problem so that this code can be merged.
> Rebase the branch
> 
> If you have a local git environment and meet the criteria below, one option is to rebase the branch and add your Signed-off-by lines in the new commits. Please note that if others have already begun work based upon the commits in this branch, this solution will rewrite history and may cause serious issues for collaborators ([described in the git documentation](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) under "The Perils of Rebasing").
> 
> You should only do this if:
> 
>     You are the only author of the commits in this branch
>     You are absolutely certain nobody else is doing any work based upon this branch
>     There are no empty commits in the branch (for example, a DCO Remediation Commit which was added using --allow-empty)
> 
> To add your Signed-off-by line to every commit in this branch:
> 
>     Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
>     In your local branch, run: git rebase HEAD~3 --signoff
>     Force push your changes to overwrite the branch: git push --force-with-lease origin add-logging-for-interop-debug

Some other useful info on DCO: https://github.com/apps/dco

File: aries_cloudagent/connections/base_manager.py (hunk 1 above)
Comment (cl0ete) [reply aries_cloudagent/connections/base_manager.py:257]: Hey I am doing some debugging on a forked repo not sure why this ended up here. I am gonna close this.

Comment (cl0ete) [reply aries_cloudagent/connections/base_manager.py:257]: Sorry about wasting your time.

**Pull request closed without merging**